packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx

```diff
@@ -67,7 +69,7 @@ export const MemberProfile = React.memo(() => {
                   <EditSymbol />
                 </EditMembershipButton>
               )}
-              <ButtonGhost size="small">
+              <ButtonGhost size="small" onClick={() => copyValue(`${window.location.host}/#/members/${member.id}`)}>
```

Comment:
I wanted to use either this or `window.location.origin`

Reply:
I think that in this case it is OK. I don't recall what's best, but host AFAIR is good enough.

Reply:
However, it looks like we should have a way to handle links globally. (ATM requires changing links in a couple of places). But that's for later.

Reply:
Moved to: #768.